package.json

```diff
@@ -3,23 +3,30 @@
   "version": "0.1.0",
   "description": "eslint plugins to support our JS Code Standards. See eslint-config-silvermine",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha 'tests/**/*.test.js'"
```

Comment:
If we're adding tests, we should add a dev dependency on a specific version of mocha. Note that we always depend on explicit/specific versions and don't use carets or tildes so as to avoid unintended upgrades.